### DISCUSSION TITLE
[@types/vimeo] not working with @vimeo/vimeo in VSCode

### DISCUSSION THREAD

**Discussion by hrvstr** [DefinitelyTyped/DefinitelyTyped#69114] [Issues with a @types package] [answered]
Hello,

I am trying to use `@types/vimeo` in my Astro project with VSCode. I have this code in my TS file:

```
import { Vimeo } from "@vimeo/vimeo";
```

And I have installed the types from this repo:

```
npm install --save @types/vimeo
added 1 package, and audited 1186 packages in 6s
```

However I still get this error on my import:

```
Could not find a declaration file for module '@vimeo/vimeo'. '/xyz/node_modules/@vimeo/vimeo/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/vimeo__vimeo` if it exists or add a new declaration (.d.ts) file containing `declare module '@vimeo/vimeo';`ts(7016)
```

Am I missing something here? How can I make VSCode recognize the types?

I tried to create a file `./vimeo.d.ts` with this content:
```
declare module "@vimeo/vimeo";
```

This makes the error go away but VSCode is not doing any type checking.

Thanks!

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
The `@types/vimeo` has types for the `vimeo` package, _not_ `@vimeo/vimeo`. The latter package doesn't have any type definitions at the moment.

**Reply by MysteryBlokHed** [reply, marked as answer]
I've opened #69141 to make a types package for the new library

**Reply by hrvstr** [reply]
Aha gotcha! Looks like I got confused by the naming. Thanks for creating the types! Really interesting system in this repo with all the bots and everything.
